dali/kernels/scratch.h

```diff
@@ -139,15 +143,22 @@ class ScratchpadAllocator {
    * @brief Returns reference to the current
    *        allocation policy for given allocation type.
    */
-  AllocPolicy &Policy(AllocType type) {
-    return buffers_[static_cast<int>(type)].policy;
-  }
+  template <typename MemoryKind>
+  AllocPolicy &Policy()             { return Policy(mm::kind2id_v<MemoryKind>); }
+
```

Comment:
This is formatting we don't use anywhere in the codebase. I also believe, that it's not consistent with itself, cause if we'd like to align signature components, we should do rather:
```c++
  template <typename MemoryKind>
        AllocPolicy &Policy()       { return Policy(mm::kind2id_v<MemoryKind>); }

  template <typename MemoryKind>
  const AllocPolicy &Policy() const { return Policy(mm::kind2id_v<MemoryKind>); }
```

I feel, that we should go here for what we already have in the codebase 
```c++
template <typename MemoryKind>
AllocPolicy &Policy() { return Policy(mm::kind2id_v<MemoryKind>); }
```

Reply:
I guess linter would scream at the former with "weird indent". I don't mind removing the extra whitespace.

Reply:
Done.